app/assets/javascripts/checkbox_control.es6

```diff
@@ -25,17 +27,32 @@ $(function() {
         Set each checkbox based on its remembered state
     */
     control.change( function() {
-      const controlChecked = this.checked
+      const controlChecked = this.checked;
+
       checkboxes.each(function(index) {
         const checkbox = $(this)
         if(controlChecked) {
-          checkboxMemory[index] = this.checked
-          checkbox.prop("checked", false)
+          checkboxMemory[index] = this.checked;
+          checkbox.prop("checked", false);
```

Comment:
Ummmm .... this is my fault. I'm a little old school and haven't got out of the habit of ending lines of JavaScript with `;`, so my original code had semicolon line terminators. Julien is much more up-to-date on these things and took the time to remove the semicolons when he touched the file. Because you've rebased, the version you are working on is Julien's sans-semicolon version .... and you've effectively put all the semicolons back in.

Julien's code is cleaner than mine. So remove the semicolons please.

Reply:
Thank you, i've removed the semi colons. 👍